lib/debug/thread_client.rb

```diff
@@ -288,22 +301,24 @@ def show_by_editor path = nil
 
     def show_locals
       if s = current_frame&.self
-        puts " %self => #{s}"
+        puts " #{colorize_cyan("%self")} => #{colored_inspect(s)}"
       end
       if current_frame&.has_return_value
-        puts " %return => #{current_frame.return_value}"
+        puts " #{colorize_cyan("%return")} => #{colored_inspect(current_frame.return_value)}"
       end
       if b = current_frame&.binding
         b.local_variables.each{|loc|
-          puts " #{loc} => #{b.local_variable_get(loc).inspect}"
+          value = b.local_variable_get(loc)
+          puts " #{colorize_cyan(loc)} => #{colored_inspect(value)}"
```

Comment:
not strong opinion:

`colorize_cyan(loc)` is needed?
If value is colored, colored name seems verbose.

![image](https://user-images.githubusercontent.com/9558/119931781-fc256c80-bfbc-11eb-84d6-ce0f3900cf35.png)

but gdb colored :)

Reply:
since we use cyan for argument names in the trace, I think it'd be consistent to color info keys the same way too.
I didn't colorize them at first, but it's kinda odd to have them white like this 🤔 

<img width="460" alt="截圖 2021-05-28 下午1 02 08" src="https://user-images.githubusercontent.com/5079556/119931922-e9a73500-bfb4-11eb-9442-3f2bb57ddfb6.png">